input/fsh/Clinical datapoints profiles/BreastCancerPatient.fsh

```diff
@@ -0,0 +1,24 @@
+Profile: BreastCancerPatient
+Parent: Patient 
+Id: ichom-patient
+Title: "Breast Cancer Patient Profile"
+Description: "The patient profiles represent the patient characteristics and demograhic factors"
+* birthDate 0..1  // is it possible to set this to datatype = date year?
+* deceased[x] only dateTime 
+* deceased[x] MS 
```

Comment:
Agree on the MS, in fact we should label all elements that we're mapping to an ICHOM variable as MS to give an indication as to what should be populated. 

Could you also populate the mappings fields to the variables we're populating - this will help us keep track of what we've done, and it'll help implementers understand how the properties relate: http://build.fhir.org/ig/HL7/fhir-shorthand/reference.html#defining-mappings

Here's an example to get you started with:

```
Mapping: breast-cancer
Id: breast-cancer
Title: "ICHOM Breast Cancer set"
Source: menopausalstatus
Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
* -> "MENOPAUSE"
``` 

It will then appear in the 'Mappings' tab